CreeDictionary/search_quality/sample.csv

```diff
@@ -1,70 +1,384 @@
+i see you,kiwâpamitin,,,,
+he sees him,wâpamêw,,,,
+she sees him,wâpamêw,,,,
+starblanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+star blanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+being taught,kiskinwahamâkosiw,,,,
+they see us,niwâpamikonânak,kiwâpamikonawak,,,
+gym,mêtawêwikamik,,,,This item and below are from NS152
+lake,sâkahikan,,,,
+juice,sîwâpoy,,,,
+hospital,âhkosîwikamik,,,,
+sock,asikan,,,,
+pail,askihk,,,,
+potato,askipwâwi,,,,
+dog,atim,nitêm,,,very irregular inflections
+doll,awâsisihkân,,,,
+tobacco,ciscêmâs,cistêmâw,,,
+fork,cîscahikanis,,,,
+girl,iskwêsis,,,,
+woman,iskwêw,,,,
+fish,kinosêw,,,,
+bacon,kohkôsiwiyin,,,,
+corn,mahtâmin,,,,
+pencil,masinahikanâhtik,,,,
+slate,masinahikanâpisk,,,,
+pen,masinahikanâpiskos,,,,
+tea,maskihkîwâpoy,,,,
+bear,maskwa,,,,
+glass,minihkwâcikan,minihkwêwiyâkanis,,,
+cup,minihkwâcikanis,,,,
+cat,minôs,,,,
+pants,mitâs,,,,
+feather,mêkwan,,,,
+road,mêskanaw,,,,
+soup,mîcimâpoy,,,,
+bead,mîkis,,,,
+knife,môhkomân,,,,
+little knife,môhkomânis,,,,
+moose,môswa,,,,
+toboggan,napakâhtik,,,,
+frying pan,napwên,,,,
+goose,niska,,,,
+boy,nâpêsis,,,,
+man,nâpêw,,,,
+Cree person,nêhiyaw,,,,
+tomato,okiniy,,,,
+little carrot,oskâcâskos,,,,
+carrot,oskâtâsk,,,,
+nut,pakân,,,,
+pepper,papêskomina,,,,
+apple,picikwâs,,,,
+ball,pâkâhtowân,,,,
+sun,pîsim,,,,
+skunk,sikâk,,,,
+sugar,sîwinikan,,,,
+salt,sîwîhtâkan,,,,
+money,sôniyâw,,,,
+butter,tohtôsâpoy,,,,
+milk,tohtôsâpoy,,,,
+fur,wayân,,,,
+dish,wiyâkan,,,,
+banana,wâkâs,,,,
+rabbit,wâpos,,,,
+egg,wâwi,,,,
+little spoon,êmihkwânis,,,,
+brings it,pêtawêw,,,,
+store,atâwêwikamik,,,,
+pig,kohkôs,,,,
+shoe,maskisin,,,,
+horse,misatim,mistatim,,,
+cow,mostos,,,,
+restaurant,mîcisowikamik,,,,
+duck,sîsîp,,,,
+bank,sôniyâwikamik,,,,
+chair,têhtapiwin,,,,
+spoon,êmihkwân,,,,
+drinks,minihkwêw,,,,
+right,tâpwê,,,,
+puppy,acimosis,,,,
+land,askiy,,,,
+child,awâsis,,,,
+skin,masakay,,,,
+fingernail,maskasiy,,,,
+hand,micihciy,,,,
+finger,micihcîs,,,,
+nipple,micohcôsimis,,,,
+face,mihkwâkan,,,,
+ear,mihtawakay,,,,
+nose,mikot,,,,
+little foot,misicis,,,,
+foot,misit,,,,
+leg,miskât,,,,
+eye,miskîsik,,,,
+arm,mispiton,,,,
+head,mistikwân,,,,
+breast,mitohtôsim,,,,
+mouth,mitôn,,,,
+body,miyaw,,,,
+tooth,mîpit,,,,
+hair,mêscakâsa,,,,
+my foot,nisit,,,,
+my head,nistikwân,,,,
+who,awîna,,,,
+oh,ka,,,,
+again,kîhtwâm,,,,
+what,kîkwây,,,,
+but,mâka,,,,
+when,tânispî,,,,
+how,tânisîyisi,,,,
+where,tânitê,,,,
+why,tânêhki,,,,
+far away,wâhyaw,,,,
+soon,wîpac,,,,
+so,êkosi,,,,
+and,êkwa,mîna,,,
+son,nikosis,,,,
+aunt,nikâwîs,,,,
+mother,nikâwiy,,,,
+grandfather,nimosôm,,,,
+husband,ninâpêm,,,,
+parents,ninêkihikwak,,,,
+younger sibling,nisîmis,,,,
+daughter,nitânis,,,,
+friend,nitôtêm,,,,
+father,nôhtâwiy,,,,
+sibling,nîtisân,,,,
+grandmother,nôhkom,,,,
+town,ôcênâsis,ôcênâs,,,
+bannock,pahkwêsikan,,,,
+duckling,sîsîpisis,,,,
+raisin,sôminis,,,,
+city,ôtênaw,,,,
+works,atoskêw,,,,
+named,isiyihkâcikâtêw,isiyihkâsow,,,
+names,isiyihkâtâw,isiyihkâtêw,,,
```

Comment:
named/names: I would expect isiyihkâsow to be the first one, and for these two to turn up the same results (though if we want to get fancy like with the plurals, seeing past tense would be cool)

Reply:
Thanks, I’ve reordered them. I think ‘isiyihkâsowina’ might apply for ‘names’ as a plural noun?

Reply:
Yes, my oversight, thanks!